developer-docs-site/docs/guides/prover-guides/spec-lang.md

```diff
@@ -1035,12 +888,12 @@ module addr::M1 {
 }
 ```
 
-When we verify `M1` standalone, the Move prover will determine that it also needs to verify
+When we verify `M1` standalone, the Move Prover will determine that it also needs to verify
 functions in `M2`, namely those which update the M2 memory such that the invariant in M1 can fail.
 
 > TODO: document mechanism and pragmas to defer invariant verification for code sections
```

Comment:
Is this TODO still relevant? Can we move to an issue for tracking instead?

Reply:
Removed because it's not relevant anymore.